server/shadow/shadow_client.c

```diff
@@ -128,6 +128,7 @@ static INLINE void shadow_client_free_queued_message(void* obj)
 
 static BOOL shadow_client_context_new(freerdp_peer* peer, rdpShadowClient* client)
 {
+	const char bind_address[] = "bind-address,";
```

Comment:
Should this maybe be all-uppercase to make its constantness and role as a sentinel more obvious? Not sure about coding style here.